Content.Server/NPC/NPCBlackboard.cs

```diff
@@ -168,7 +198,53 @@ private bool TryGetEntityDefault(string key, [NotNullWhen(true)] out object? val
                 var blocker = entManager.EntitySysManager.GetEntitySystem<ActionBlockerSystem>();
                 value = blocker.CanMove(owner);
                 return true;
+            }
+            case FreeHands:
+            {
+                if (!TryGetValue(Owner, out owner, entManager) ||
+                    !entManager.TryGetComponent<HandsComponent>(owner, out var hands) ||
+                    hands.ActiveHand == null)
+                {
+                    return false;
+                }
+
+                var handos = new List<string>();
+
+                foreach (var (id, hand) in hands.Hands)
+                {
+                    if (!hand.IsEmpty)
+                        continue;
+
+                    handos.Add(id);
+                }
+
+                value = handos;
+                return true;
+            }
+            case Inventory:
+            {
+                if (!TryGetValue(Owner, out owner, entManager) ||
+                    !entManager.TryGetComponent<HandsComponent>(owner, out var hands) ||
+                    hands.ActiveHand == null)
+                {
+                    return false;
+                }
+
+                var handos = new List<string>();
+
+                foreach (var (id, hand) in hands.Hands)
+                {
+                    if (!hand.IsEmpty)
+                        continue;
+
+                    handos.Add(id);
+                }
+
+                value = handos;
+                return true;
+            }
```

Comment:
this whole switch is starting to look suspiciously like copy paste, can this be deduplicated somehow?